PirateGame/Assets/Scripts/UserShipAgent.cs

```diff
@@ -94,6 +90,7 @@ void Update ()
 			//underneath the messages for maxed upgrades and no loot.
 
 		}
+		//TODO: should be able to move in waiting state so you can move around the island, not just get stopped on one side
```

Comment:
I makes a difference in the situation that they didn't actually want to stop at the island but are forced to because they hit the collider.  In my opinion, they should be only stopped after they hit space to "dock".  Something also to bring up in the meeting today.